cli/internal/helm/charts/edgeless/operators/charts/constellation-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
```

Comment:
Having one helmignore in the top-level chart:
- package subchart without dedicated helmignore --> pkg includes files ignored in the top-level chart
- package top-level chart --> subchart will doesn't include ignored files

Should I still remove the subchart helmignores? I think both is fine.

Reply:
If we want to package the subcharts individually it might be worth to have a helmignore in each of them.
I think its fine to keep them at least until we have a full decision on how we plan to implement helm chart updates